src/server.js

```diff
@@ -53,7 +50,7 @@ server.setRequestHandler(CallToolRequestSchema, request => {
       throw new Error('Business name and description are not provided')
     }
 
-    if (toolName === 'b12_website_generator') {
+    if (toolName === 'generate_website') {
```

Comment:
I looked at tool names for other extensions and they all started with verbs.

Reply:
<img width="709" height="790" alt="Screenshot 2025-07-28 at 12 40 30 PM" src="https://github.com/user-attachments/assets/6eda62ee-275e-453d-a850-49e42fad33db" />